CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Ansible Modules for HPE OneView Change Log
 
+## v5.6.0(unreleased)
+This release extends the planned support of the modules to OneView REST API version 800 (OneView v4.1), 1000 (OneView v4.2) and 1200 (OneView v5.0) and 1600(Oneview v5.2).
```

Comment:
Write it as 1000(OneView v4.2), 1200 (OneView v5.0) and 1600(Oneview v5.2)

Reply:
done